x-pack/plugins/observability_solution/dataset_quality/server/routes/register_routes.ts

```diff
@@ -39,9 +39,12 @@ export function registerRoutes({
   const router = core.http.createRouter();
 
   routes.forEach((route) => {
-    const { endpoint, options, handler, params } = route;
+    const { endpoint, handler } = route;
```

Comment:
Do you need to add the `security` passing here too?

Reply:
yes good catch - we should make an effort to migrate all plugins to use `registerRoutes` from the package, it's become too complicated to leave up to the consumer.

Reply:
See https://github.com/elastic/kibana/issues/198681

We're working on some pre-reqs as part of https://github.com/elastic/kibana/issues/198684 with the DevEx WG's resources but then we should be able to request owning teams to migrate